code/game/turfs/simulated/reebe_void.dm

```diff
@@ -0,0 +1,47 @@
+/turf/open/indestructible/reebe_void
+	name = "void"
+	icon_state = "reebemap"
+	layer = SPACE_LAYER
+	baseturf = /turf/open/indestructible/reebe_void
+	planetary_atmos = TRUE
+
+/turf/open/indestructible/reebe_void/Initialize(mapload)
+	. = ..()
+	icon_state = "reebegame"
+
+/turf/open/indestructible/reebe_void/spawning
+	icon_state = "reebespawn"
+
+/turf/open/indestructible/reebe_void/spawning/Initialize(mapload)
+	. = ..()
+	if(mapload)
+		for(var/i in 1 to 3)
+			if(prob(1))
+				new /obj/item/clockwork/alloy_shards/large(src)
+			if(prob(2))
+				new /obj/item/clockwork/alloy_shards/medium(src)
+			if(prob(3))
+				new /obj/item/clockwork/alloy_shards/small(src)
+
+/turf/open/indestructible/reebe_void/spawning/lattices
+	icon_state = "reebelattice"
+
+/turf/open/indestructible/reebe_void/spawning/lattices/Initialize(mapload)
+	. = ..()
+	if(mapload)
+		if(prob(2.5))
+			new /obj/structure/lattice/catwalk/clockwork(src)
+		else if(prob(5))
+			new /obj/structure/lattice/clockwork(src)
+
+/turf/open/indestructible/reebe_void/Enter(atom/movable/AM, atom/old_loc)
+	if(!..())
+		return FALSE
```

Comment:
You might want to replace these with just `return`s, though.

Reply:
it's already written, so no.

Reply:
Fuck off Joan

Reply:
Why would having to make another commit ever justify improper code? When has that ever been a thing?

Reply:
It's not even improper code, just technically unnecessary.

Reply:
I can't understand your logic here. It's the standard to use normal `return` statements instead of `return FALSE`. I don't think it makes sense to just pick and choose when to obey those standards.

Reply:
This is not a code standard, and thus the literal definition of bikeshedding.

Reply:
objectively, a naked return is not the same as return FALSE